Laptop Price Predictor/Model/Laptop_Price_Prediction.py

```diff
@@ -0,0 +1,62 @@
+# LAPTOP PRICE PREDICTION
+
+# Importing Libraries
+
+import pandas as pd
+from sklearn import preprocessing
+from statsmodels.stats.outliers_influence import variance_inflation_factor
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.metrics import r2_score
+
+# Getting our Data
+
+df = pd.read_csv(r'./Laptop Price Predictor/Dataset/laptoppricing.csv', encoding = 'unicode_escape')
+
+# Data Preprocessing
+
+# checking for null values
+print('\n',df.isnull().any())
+
+print('\n',df.columns)
+
+label_encoder = preprocessing.LabelEncoder()
+df['ï»¿Manufacturer'] = df['ï»¿Manufacturer'].astype('|S')
```

Comment:
what is this character?: `ï»¿`Manufacturer 

if the column is not clean, then add the step to clean it